class/defaults.yml

```diff
@@ -1,23 +1,16 @@
 parameters:
-  grafana:
+  grafana_helm:
     namespace: syn-grafana
     charts:
       grafana: 6.1.16
-    images:
-      grafana:
-        image: grafana/grafana
-        tag: ' 7.3.5'
-        pullPolicy: IfNotPresent
-      downloadDashboardsImage:
-        image: curlimages/curl
-        tag: 7.73.0
-    serviceMonitor:
-      enabled: true
-    ingress:
-      enabled: false
-      hosts:
-      - chart-example.local
-      tls: []
-      annotations: {}
-
-
+    helm_values:
+      images:
+        grafana:
+          image: grafana/grafana
+          tag: ' 7.3.5'
```

Comment:
```suggestion
          tag: '7.3.5'
```

Reply:
I think we can just remove the settings regarding the image here and use the defaults?